### PR TITLE
[MTDSA-11720] Json Example for Retrieve Tax Calc RAML

### PR DESCRIPTION
**Pull request by Mbrookesy**

### CODE REVIEW

**Review by narendravijayarao — APPROVED** [review]

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 1]
```diff
@@ -1,225 +1,228 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "A statement summarising the profit and loss position for business income sources. The enum in the incomeSourceTypes will determine the business type. When the type is \"self-employment\" totalExpenses and totalAdditions may be negative amounts. Other business sources will continue to return positive amounts.",
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:3]: The description does not match the spec

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 2]
```diff
@@ -1,225 +1,228 @@
...
+        "type": "string"
+      },
+      "totalIncome": {
+        "title": "totalIncome",
+        "description": "The total income used in the business calculation. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": 0,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:39]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 3]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "totalExpenses": {
+        "title": "totalExpenses",
+        "description": "The total expenses used in the business calculation. This value must be between -99999999999.99 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": -99999999999.99,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:48]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 4]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "netProfit": {
+        "title": "netProfit",
+        "description": "The net profit for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": 0,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:57]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 5]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "netLoss": {
+        "title": "netLoss",
+        "description": "The net loss for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": 0,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:66]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 6]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "totalAdditions": {
+        "title": "totalAdditions",
+        "description": "The total additions to the net profit (or deductions from the net loss) for the business. This value must be between -99999999999.99 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": -99999999999.99,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:75]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 7]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "totalDeductions": {
+        "title": "totalDeductions",
+        "description": "The total deductions from the net loss (or additions to the net profit) for the business. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": 0,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:84]: The example should be a decimal

File: resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json [hunk 8]
```diff
@@ -1,225 +1,228 @@
...
+      },
+      "accountingAdjustments": {
+        "title": "accountingAdjustments",
+        "description": "Accounting adjustments applied to the profit or loss calculation for Foreign Property. This value must be between -99999999999.99 and 99999999999.99 up to 2 decimal places.",
+        "type": "number",
+        "multipleOf": 0.01,
+        "minimum": -99999999999.99,
+        "maximum": 99999999999.99,
+        "example": "160"
```
Comment (vfryer12) [inline resources/public/api/conf/3.0/schemas/retrieve/calculation/businessProfitAndLoss.json:93]: The example should be a decimal

**Pull request merged**